README.md

```diff
@@ -10,6 +10,7 @@ To add a conference to the list, please [contribute](#contributing)!
 |------------|------|-------|-------------|
 | [JSFoo](https://jsfoo.in/2018/) | 13-14 Sep | Bengaluru | The eighth edition of India’s first JavaScript conference.
 | [PyCon India](https://in.pycon.org/2018/) | 22-26 Aug | Bengaluru International Exhibition Center (BIEC), Bengaluru | The premier conference in India on using and developing the Python programming language. |
+| [Deccan RubyConf 2018](http://www.deccanrubyconf.org/) | 4 Aug | Hotel Sheraton Grand, Bund Garden Hotel, Sangamvadi, Pune | The fifth edition of the regional single track ruby conference + workshops. Tickets can be purchased at https://www.townscript.com/e/drc-2018 |
```

Comment:
For the description, please remove the information about the tickets. Only the first line should be sufficient.

Reply:
The website says that the date is 12 Aug - http://www.deccanrubyconf.org/index.html

Reply:
Oh that's for 2017. Looks like the information for 2018 is not published yet. I'd like to hold this PR until the information is available officially.

Reply:
They have not updated the official website, but have given the date on the purchase tickets page. But its fine if we hold this till the website is updated as well. 👍